materialize/resources/resource_materialized_view.go

```diff
@@ -0,0 +1,241 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmoiron/sqlx"
+)
+
+var materializedViewSchema = map[string]*schema.Schema{
+	"name": {
+		Description: "The identifier for the materialized view.",
+		Type:        schema.TypeString,
+		Required:    true,
+	},
+	"schema_name": {
+		Description: "The identifier for the materialized view schema.",
+		Type:        schema.TypeString,
+		Optional:    true,
+		Default:     "public",
+		ForceNew:    true,
+	},
+	"database_name": {
+		Description: "The identifier for the materialized view database.",
+		Type:        schema.TypeString,
+		Optional:    true,
+		Default:     "materialize",
+		ForceNew:    true,
+	},
+	"qualified_name": {
+		Description: "The fully qualified name of the materialized view.",
+		Type:        schema.TypeString,
+		Computed:    true,
+	},
+	"in_cluster": {
+		Description: "The cluster to maintain this materialized view.",
```

Comment:
Should we add some info about the defaults for this? Either in the form of a default value or just in the description that it will be in the default cluster if not selected.

Reply:
Just updated the description, let me know what you think